apps/src/lib/kits/maker/commands.js

```diff
@@ -6,12 +6,12 @@ import {
 import {BOARD_EVENT_ALIASES} from './boards/circuitPlayground/PlaygroundConstants';
 import MicroBitBoard from './boards/microBit/MicroBitBoard';
 
-/** @private {CircuitPlaygroundBoard} */
+/** @private {CircuitPlaygroundBoard, MicroBitBoard} */
```

Comment:
I would like to confirm this is the correct syntax for this private variable.

Reply:
@sanchitmalhotra126 lmk what you think.

Reply:
Hmm good question, I don't know if we have a good standard for union types like this in plain JavaScript...I think `{CircuitPlaygroundBoard | MicroBitBoard}` (borrowing [TypeScript union type syntax](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types)) would make the most sense to me at least?

Reply:
Thank you! This makes sense. Will use.